File: tests/fixtures/negative/security/RuntimeExecUnsafeShape.java
package test;

class RuntimeExecUnsafeShapeJavaSafeFixture {
    void list(String userPath) throws java.io.IOException {
        Runtime.getRuntime().exec(new String[] {"ls", "-la", userPath});
        Runtime.getRuntime().exec("ls -la");
        new ProcessBuilder("ls", "-la", userPath).start();
    }
}

File: tests/fixtures/negative/security/RuntimeExecUnsafeShape.kt
package test

class RuntimeExecUnsafeShapeSafeFixture {
    fun list(userPath: String) {
        Runtime.getRuntime().exec(arrayOf("ls", "-la", userPath))
        Runtime.getRuntime().exec("ls -la")
        ProcessBuilder("ls", "-la", userPath).start()
    }
}
